docs/topics/email.txt

```diff
@@ -415,6 +420,21 @@ Django's email library, you can do this using the
             msg.attach_alternative(html_content, "text/html")
             msg.send()
 
+    .. attribute:: alternatives
+
+        A list of named tuples with attributes ``(content, mimetype)``. This is
+        particularly useful in tests::
+
+            self.assertEqual(len(msg.alternatives), 1)
+            self.assertEqual(msg.alternatives[0].content, html_content)
```

Comment:
```suggestion
            self.assertEqual(msg.alternatives[0].mime_type, "text/html")
```
Note that we've documented it to be `mimetype` rather than `mime_type`, I _think_ we prefer `mime_type` and should update the docs?

I also think adding tests doesn't hurt as at the very least it's easier to spot if we document the names wrong if we're using them in tests
```diff
--- a/tests/mail/tests.py
+++ b/tests/mail/tests.py
@@ -550,6 +550,29 @@ class MailTests(HeadersCheckMixin, SimpleTestCase):
         msg.attach("example.txt", "Text file content", "text/plain")
         self.assertIn(html_content, msg.message().as_string())

+    def test_attachments(self):
+        msg = EmailMessage()
+        file_name = "example.txt"
+        file_content = "Text file content"
+        mime_type = "text/plain"
+        msg.attach(file_name, file_content, mime_type)
+        self.assertEqual(msg.attachments[0][0], file_name)
+        self.assertEqual(msg.attachments[0].filename, file_name)
+        self.assertEqual(msg.attachments[0][1], file_content)
+        self.assertEqual(msg.attachments[0].content, file_content)
+        self.assertEqual(msg.attachments[0][1], mime_type)
+        self.assertEqual(msg.attachments[0].mime_type, mime_type)
+
+    def test_alternatives(self):
+        msg = EmailMultiAlternatives()
+        html_content = "<p>This is <strong>html</strong></p>"
+        mime_type = "text/html"
+        msg.attach_alternative(html_content, mime_type)
+        self.assertEqual(msg.alternatives[0][0], html_content)
+        self.assertEqual(msg.alternatives[0].content, html_content)
+        self.assertEqual(msg.alternatives[0][1], mime_type)
+        self.assertEqual(msg.alternatives[0].mime_type, mime_type)
+
     def test_none_body(self):
```

I also can't accept tickets I create, so feel free to accept this ticket: https://code.djangoproject.com/ticket/35537

Reply:
A quick search through the codebase suggests `mimetype` is more common than `mime_type`, although there isn't much consistency.

Definite +1 for adding tests though - I'll get those added!

Reply:
Let's go for `mimetype` then 👍